main.js

```diff
@@ -0,0 +1,110 @@
+const $ = (selector) => document.querySelectorAll(selector);
+
+const burger = $(".burger__card");
+const totalPrice = document.querySelector(".finalPrice__value");
+const orderCancel=document.querySelector('.modal__cancel');
+const orderBtn=document.querySelector('.cart__orderBtn');
+const allCancel=document.querySelector('.cart__cancelBtn');
+const cartArea = document.querySelector(".cart__title");
+function burgerClick(idx) {
+  burger[idx].addEventListener("click", (e) => {
+    cart_item(idx);
+  });
+}
```

Comment:
```suggestion
function burgerClick(idx) {
  burger[idx].addEventListener("click", () => {
    cart_item(idx);
  });
}
```

이벤트 객체는 안 쓰고 있으니까 지워주세용!